modules/spring-config/src/main/java/org/mule/config/spring/SpringRegistryLifecycleManager.java

```diff
@@ -7,20 +7,41 @@
 package org.mule.config.spring;
 
 import org.mule.api.MuleContext;
+import org.mule.api.agent.Agent;
+import org.mule.api.component.Component;
+import org.mule.api.config.Config;
+import org.mule.api.construct.FlowConstruct;
+import org.mule.api.exception.MessagingExceptionHandler;
 import org.mule.api.lifecycle.Disposable;
 import org.mule.api.lifecycle.Initialisable;
+import org.mule.api.lifecycle.LifecycleCallback;
 import org.mule.api.lifecycle.LifecycleException;
-import org.mule.api.lifecycle.LifecyclePhase;
 import org.mule.api.lifecycle.Startable;
 import org.mule.api.lifecycle.Stoppable;
+import org.mule.api.model.Model;
 import org.mule.api.registry.Registry;
+import org.mule.api.routing.OutboundRouter;
+import org.mule.api.routing.OutboundRouterCollection;
+import org.mule.api.source.MessageSource;
+import org.mule.api.store.ObjectStoreManager;
+import org.mule.api.transformer.Transformer;
+import org.mule.api.transport.Connector;
+import org.mule.context.notification.ServerNotificationManager;
 import org.mule.lifecycle.EmptyLifecycleCallback;
+import org.mule.lifecycle.LifecycleObject;
+import org.mule.lifecycle.NotificationLifecycleObject;
 import org.mule.lifecycle.RegistryLifecycleManager;
-import org.mule.lifecycle.phases.ContainerManagedLifecyclePhase;
+import org.mule.lifecycle.phases.MuleContextDisposePhase;
+import org.mule.lifecycle.phases.MuleContextInitialisePhase;
 import org.mule.lifecycle.phases.MuleContextStartPhase;
 import org.mule.lifecycle.phases.MuleContextStopPhase;
 import org.mule.lifecycle.phases.NotInLifecyclePhase;
+import org.mule.processor.AbstractMessageProcessorOwner;
 import org.mule.registry.AbstractRegistryBroker;
+import org.mule.routing.requestreply.AbstractAsyncRequestReplyRequester;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
 
 public class SpringRegistryLifecycleManager extends RegistryLifecycleManager
```

Comment:
I'm unsure, with the rest of the changes, why a spring specific lifecycle manager is needed at all now?